app/models/miq_queue_worker_base/runner.rb

```diff
@@ -93,7 +93,7 @@ def get_message
 
   def message_delivery_suspended?
     if self.class.require_vim_broker?
-      return true unless MiqVimBrokerWorker.available?
+      return true unless ManageIQ::Providers::Vmware::InfraManager::VimBrokerWorker.available?
     end
```

Comment:
Only the VMware `MetricsCollectorWorker::Runner` and `RefreshWorker::Runner` use `require_vim_broker?` so maybe this whole thing should be ripped out and `message_deliver_suspended?` can be implemented by those classes?